lib/ice_cube/rule.rb

```diff
@@ -73,6 +73,7 @@ class << self
       # Convert from a hash and create a rule
       def from_hash(original_hash)
         hash = IceCube::FlexibleHash.new original_hash
+        raise ArgumentError, 'empty ical rule_type' if hash[:rule_type].nil?
```

Comment:
I think we can do something more here by saying:

`raise ArgumentError, 'Invalid rule type' unless hash[:rule_type] && hash[:rule_type].match(/\:\:(.+?)Rule/)`